_playwright-tests/style_guide.md

```diff
@@ -0,0 +1,238 @@
+# Playwright Style Guide 💅
```

Comment:
Can you add link to README.md to this guide? There is PW section so link if from there?